README.md

```diff
@@ -52,6 +52,48 @@ npm install -D webworkify
 
 See the [webpack.js](webpack.js) and [browserify.js](browserify.js) files for examples as to how one might configure for a different bundler.
 
+### Bundler Performance
+
+Using `physijs-webpack` may bulk up your bundle by quite a bit, primarily because `physijs` depends on `ammo.js`, which is roughly ~1.2MB (minified) alone.
+
+While caching of repeated builds will certainly speed up performance by quite a bit, we can use some other optimizations to further improve your bundler's performance.
+One of these is the `noParse` config option ([webpack](https://webpack.js.org/configuration/module/#modulenoparse)|[browserify](https://github.com/browserify/browserify#usage)) that instructs the bundler to skip parsing of certain files and is usually used as an optimization for large files (like `ammo.js`!).
+
+With `webpack`, usage may look like:
```

Comment:
Ok, added a bold "Please note" at the top stating that is optional, not necessary, and only for advanced users.

I also added "optimizations" to the heading to make it more clear that it's optional (as optimizations are not generally required).

I think making this an H2 instead of an H3 would make the separation between optional and required more obvious, but this _is_ part of the _usage_ of the library... hmm...
Another improvement might be sectioning out `#### noParse` and `#### other optimizations`

Reply:
ok, did that, and I do think it's a bit better, but I'm still not a huge fan. It also makes the library _seem_ harder to use, bc bigger docs and webpack config, even if this is optional... maybe it should go in a separate file and just be linked to from the README?

Reply:
Split it out into its own file with the latest commit added to this PR, I like it. The README is small so it's still easily noticeable, though it may be glanced over when returning users come back and read through, but not much I can do on that front 😕 . Maybe link the entire heading as well so it's in blue (and therefore more noticeable)?

Reply:
Linked the entire heading. I'm satisfied now, marking as resolved -- let's get this merged and published!